src/main/java/org/jitsi/jibri/RecordingSinkType.java

```diff
@@ -27,7 +27,11 @@
 public enum RecordingSinkType
 {
     STREAM("stream"),
-    FILE("file");
+    FILE("file"),
+    //TODO: putting gateway in here doesn't feel great (and isn't what the xmpp uses anyway).
+    // we need some top-level param that denotes what we're doing (recording a file,
+    // streaming to youtube, or doing a sipgateway)
+    GATEWAY("gateway");
```

Comment:
i'm gonna come back around in the http api (which is the only thing relying on this for now) and change it to use different http endpoints per service (instead of 1 startService endpoint) which will make fixing this easier.